roles/kubernetes/master/templates/kubeadm-config.v1alpha3.yaml.j2

```diff
@@ -221,39 +221,38 @@ schedulerExtraVolumes:
 ---
 apiVersion: kubeproxy.config.k8s.io/v1alpha1
 kind: KubeProxyConfiguration
-bindAddress: 0.0.0.0
+bindAddress: {{ kube_proxy_bind_address }}
 clientConnection:
- acceptContentTypes: ""
- burst: 10
- contentType: application/vnd.kubernetes.protobuf
- kubeconfig: /var/lib/kube-proxy/kubeconfig.conf
- qps: 5
-clusterCIDR: ""
-configSyncPeriod: 15m0s
+ acceptContentTypes: {{ kube_proxy_client_accept_content_types }}
```

Comment:
You should add `"` to all the vars because some of them must be strings

Reply:
The yaml parsing should already be interpreting string in the template where required, as long as users follow the guidelines/defaults as examples, their overrides should be handled properly.

However, it's late and I might be missing something. Do you have an example that would cause the types would mismatch?

At some point in the future I'd like to add some prechecks for user facing variable type validation across the project. We should be able to write some code generation or leverage kubeadm's check modes to automate validation code (and other generated code) for all of the component configs.